CHANGES.rst

```diff
@@ -43,7 +43,8 @@ astropy.io.fits
 - Added ``ver`` attribute to set the ``EXTVER`` header keyword to ``ImageHDU``
   and ``TableHDU``. [#6454]
 
-- The performance for reading FITS tables has been significantly improved, in
+- The performance for reading FITS tables has been significantly improved
+  for cases where the table contains one or more string columns, in
```

Comment:
I think it's useful to mention that the best improvement is for string columns, thanks, but it's not only for string columns. What about "... significantly improved *in particular* for cases where ..." or something similar ?

Reply:
I agree - fixed :)